Octokit/Http/Connection.cs

```diff
@@ -238,12 +238,12 @@ public Task<IApiResponse<T>> Post<T>(Uri uri)
             return SendData<T>(uri, HttpMethod.Post, null, null, null, CancellationToken.None);
         }
 
-        public Task<IApiResponse<T>> Post<T>(Uri uri, object body, string accepts, string contentType)
+        public Task<IApiResponse<T>> Post<T>(Uri uri, object body, string accepts, string contentType, IDictionary<string, string> parameters = null)
```

Comment:
since this is an existing public method, we want to avoid making a breaking change if possible, so typically we would add another method with the new signature